numpy/core/tests/test_scalarprint.py

```diff
@@ -199,6 +200,61 @@ def test_dragon4_interface(self):
                          "1.2" if tp != np.float16 else "1.2002")
             assert_equal(fpos(tp('1.'), trim='-'), "1")
 
+    @dec.knownfailureif(not platform.machine().startswith("ppc64"))
+    def test_ppc64_ibm_double_double128(self):
+        # check that the precision decreases once we get into the subnormal
+        # range. Unlike float64, this starts aroung 1e-292 instead of 1e-308,
+        # which happens when the first double is normal and the second is
+        # subnormal.
+        # Note: We construct these values by division because many
+        # versions of glibc have a bug in strtold that they fail to get
+        # float128 values from a string if in the subnormal range.
+        x = np.float128('2.123123123123123123123123123123123e-286')
+        got = [str(x/np.float128('2e' + str(i))) for i in range(0,40)]
```

Comment:
Does `float128(2**i)` work here, or did I never finish implementing that?

Reply:
No, not in master. 

Also, the comment is wrong: strtold is giving the right result, and also correctly setting errno. I'll just remove the comment.